samples/routing/fair-distribution/Core_6/Client/Program.cs

```diff
@@ -83,9 +83,9 @@ static void AddRouting(EndpointConfiguration endpointConfiguration)
         #region Routing
 
         const string server = "Samples.FairDistribution.Server";
-        var routing = endpointConfiguration.Routing();
+        var routing = endpointConfiguration.UseTransport<MsmqTransport>().Routing();
         routing.RouteToEndpoint(typeof(PlaceOrder), server);
-        routing.Mapping.Physical.Add(
+        routing.GetSettings().GetOrCreate<EndpointInstances>().Add(
```

Comment:
-1 to any `.Advanced.` api. the fact of an api being advanced or not is only relative to the consumer. their experience level will be the determining factor on if an api seems advanced.

also there are many other apis in NSB that would also classify as "advanced" based on the above logic. we will just end up polluting people code with verbose APIs in an effort to optimize for people who dont know when to not hard code a machine name. 

This should be handled with documentation. not with changes to the API that will make it more complex for people who know what they are doing